docker-compose-io.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
+services:
+  app-pgstac:
+    container_name: stac-fastapi-pgstac
+    build: .
+    platform: linux/amd64
+    environment:
+      - APP_HOST=0.0.0.0
+      - APP_PORT=8082
+      - RELOAD=true
+      - ENVIRONMENT=production
+      - POSTGRES_USER=${PGUSER}
+      - POSTGRES_PASS=${PGPASSWORD}
+      - POSTGRES_DBNAME=${PGDATABASE}
+      - POSTGRES_HOST_READER=${PGHOST}
```

Comment:
if `PGHOST` is different from the database container name it fails, so better not to keep it as an env var